ui/src/components/ClubsCarousel/index.tsx

```diff
@@ -17,15 +18,9 @@ export const CLUB_OPTIONS = [
 
 Object.freeze(CLUB_OPTIONS);
 
-interface ClubsCarouselData {
-  name: string;
-  logo: string;
-  category: string;
-}
-
 export const ClubsCarousel: React.FC = () => {
   const [selectedOption, setSelectedOption] = useState(CLUB_OPTIONS[0]);
-  const {data, error} = useAuthFetch('/clubs');
+  const {data, error} = useAuthFetch<{results: ClubData[]}>('/clubs');
   const clubsData = data ? data.results : null;
```

Comment:
This is a paginated response, correct?

We can create a `PaginatedResponse<T>` interface, which will contain all the generic paginated related fields as well as an array of Ts.

Reply:
What is the type of `next` & `previous` here?
```
count: 3
next: null
previous: null
results: (3) [{…}, {…}, {…}]
```

Reply:
they are strings